golem-worker-executor-base/src/grpc.rs

```diff
@@ -852,6 +852,7 @@ impl<Ctx: WorkerCtx, Svcs: HasAll<Ctx> + UsesAllDeps<Ctx = Ctx> + Send + Sync +
                 while let Ok(item) = receiver.recv().await {
                     match item {
                         worker_event::WorkerEvent::Close => {
+                            debug!("WORKER_EVENT_STREAM CLOSED");
```

Comment:
we should add worker id to log

Reply:
Ah, I did not want to leave this log in it it was just for debugging